pkgs/games/prismlauncher/default.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  jdk8,
+  jdk,
+  zlib,
+  file,
+  wrapQtAppsHook,
+  xorg,
+  libpulseaudio,
+  qtbase,
+  libGL,
+  quazip,
+  glfw,
+  openal,
+  extra-cmake-modules,
+  tomlplusplus,
+  ghc_filesystem,
+  msaClientID ? "",
+  jdks ? [jdk jdk8],
```

Comment:
Why do we need both?

Reply:
Different MC versions and packs require different versions, JDK8 is common for old ones and current is common for new so it's a reasonable default.

Reply:
We likely don't need both jdks, this was based on the PolyMC derivation which had both jdks. I'm going to try removing one and checking if it still works fine

Reply:
That will make launching old packs significantly less convenient.

Reply:
Removing `jdk8` works fine for any version newer than Minecraft 1.17. But once you want to play something older, you will need `jdk8`. Same problem other way around